content/about/open-letter-meta-dma/technical-briefing.md

```diff
@@ -0,0 +1,271 @@
+
+In **2009**, **Google placed XMPP at the core of its collaborative editing tool, Google Wave**, further demonstrating the protocol’s adaptability beyond instant messaging. Meanwhile, **social networks across the world embraced XMPP**, with notable implementations by **Yandex in Russia** and **StudiVZ in Germany**.
+
+### A Proven Standard for Messaging Interoperability
+
+XMPP has played a crucial role in shaping the modern messaging landscape. Its success demonstrates that **true interoperability is achievable through open, federated protocols**, and that major technology companies—including Google and Facebook—have previously recognized its value. As the DMA now mandates interoperability for messaging services, XMPP remains the most viable and battle-tested solution to meet these requirements.
+
+## Smartphones, WhatsApp, and the Gatekeeping Strategy
+
+Despite XMPP’s success, technical advantages, and suitability for messaging interoperability, its adoption slowed as the messaging landscape evolved. The shift toward proprietary platforms coincided with the rise of smartphones, which led to growing financial stakes in messaging services and increased efforts by dominant players to consolidate control over their networks.
+
+### The Shift to Mobile and the Rise of Proprietary Networks
+
+In **2007**, Apple launched the iPhone, ushering in the smartphone era and the widespread adoption of mobile internet. By **2009**, the introduction of both the App Store and push notification services paved the way for the launch of **WhatsApp**. Messaging, which had previously been limited to SMS on mobile, quickly became a **rich and ubiquitous communication tool**.
+
+WhatsApp took a strategic approach by leveraging a public resource—the phone number system—as a unique user identifier. Combined with easy access to users’ phone contacts in modern smartphone operating systems, WhatsApp was able to build an extensive **proprietary** messaging network. In essence, it **became the new ICQ**, but at an even larger scale. As the service grew, it positioned itself **in direct competition with mobile carriers**, as **over-the-top (OTT) messaging**—services that bypass traditional carrier networks by leveraging the open internet. At the time, **net neutrality** was seen as a safeguard ensuring that carriers could not abuse their control over communication infrastructure, but it also allowed closed proprietary messaging services to thrive.
```

Comment:
I think you want to mention here that WhatsApp started as an XMPP service with a locked-in client. (Although it is mentioned later).

Reply:
I added a clause in [cd46751](https://github.com/xsf/xmpp.org/pull/1495/commits/cd46751edf74f61167cbaf2dbf3f523f85db558a).